docker-compose.yaml

```diff
@@ -7,5 +7,11 @@ services:
     working_dir: /go/src/github.com/ismaelchess/go-share-secret
     volumes: 
       - .:/go/src/github.com/ismaelchess/go-share-secret
+    links:
+      - redis:devredis1      
     ports: 
-    - 8080:8080
\ No newline at end of file
+      - 8080:8080
+
+  redis:
+    build: ./dockerfiles/redis
```

Comment:
This is ok, but it won't store permanently redis data. After a restart you will loose everything